src/cattrs/gen/typeddicts.py

```diff
@@ -234,6 +234,7 @@ def make_dict_structure_fn(
     _cattrs_forbid_extra_keys: bool | Literal["from_converter"] = "from_converter",
     _cattrs_use_linecache: bool = True,
     _cattrs_detailed_validation: bool | Literal["from_converter"] = "from_converter",
+    _cattrs_use_alias: bool | Literal["from_converter"] = "from_converter",
```

Comment:
I'm wondering: is there also some change for `cattrs.gen.typeddicts.make_dict_unstructure_fn` needed?